docs/DEVELOP.adoc

```diff
@@ -180,33 +180,24 @@ By default the `Makefile` runs all of it's command in a clean Docker container,
 
 The CI pipeline also uses the `Makefile`, so if all commands in the `Makefile` succeed locally, the CI is most likely to pass as well.
 
-=== Build and install with setup.py
+=== Build and install with pyproject-build
 
 To install, simply run:
 
  pip install .
 
 TIP: if pip isn't present on your system, or too old, you can install or upgrade it with `python -m ensurepip --upgrade`
 
-The build process can be also be run separately:
+The build process can be also be run separately using `pyproject-build`.
 
- python3 setup.py build
-
-The setup script expects to find librsync headers and libraries in the default location, usually /usr/include and /usr/lib.
-If you want the setup script to check different locations, use the --librsync-dir switch or the LIBRSYNC_DIR environment variable.
+The setup.py script expects to find librsync headers and libraries in the default location, usually `/usr/include` and `/usr/lib`.
+If you want the setup script to check different locations, use the LIBRSYNC_DIR environment variable.
 For instance to instruct the setup program to look in `/usr/local/include` and `/usr/local/lib` for the librsync files run:
 
- python3 setup.py --librsync-dir=/usr/local build
+ LIBRSYNC_DIR=/usr/local python3 setup.py build
```

Comment:
Did you forget this one ?

Reply:
Possibly but it's still a draft and I know that I'm not finished :smirk: 
Seriously, the developer documentation needs a serious refresh with proper testing, because `./setup.py build` doesn't really have an equivalent in pyproject-build, so it's not only a question of replacing the command, the whole approach needs to be adapted.

Reply:
I was looking at your pull request by chance. I didn't even know `pyproject-build` existed.